mypy/checker.py

```diff
@@ -1758,9 +1758,6 @@ def visit_try_stmt(self, s: TryStmt) -> Type:
         self.binder.try_frames.add(len(self.binder.frames) - 2)
         self.accept(s.body)
         self.binder.try_frames.remove(len(self.binder.frames) - 2)
-        if s.else_body:
-            self.accept(s.else_body)
-        self.breaking_out = False
```

Comment:
I suspect that `self.breaking_out = False` should be left here. I can try to come up with a test case, but if nothing breaks, I'd leave it.